src/remarx/polar_utils.py

```diff
@@ -0,0 +1,118 @@
+import csv
+from collections import defaultdict
+
+import polars as pl
+
+
+def load_title_phrases(filename: str) -> dict[str, str]:
+    """
+    Load title phrases from file. Returns a dictionary mapping
+    a title (e.g., "Kapital") to a list of its search phrases
+    used for identifying candidate sentences.
+    """
+    title_phrases = defaultdict(list)
+    with open(filename, encoding="utf-8-sig", newline="") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            title_phrases[row["title"]].append(row["phrase"])
+    return title_phrases
+
+
+def load_candidate_sentences(
+    filename: str, title_phrases: dict[str, str]
+) -> pl.DataFrame:
+    """
+    Load from file and prepare candidate sentences dataframe.
```

Comment:
Suggesting a bit more detail for context of what goes in / what is expected out, based on my test run of this code in a notebook. Adapt as useful.

```suggestion
    Load candidate sentences from a CSV file and prepare a dataframe for combination with other data. Expects a CSV file generated by `candidate_mentions` script. Constructs a sentence id field and adds fields indicating which titles each sentence is a candidate for, based on the title phrase lookup passed in.
```